<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Qt-DAB</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="appName">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Qt-DAB radio</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignmentFlag::AlignHCenter">
    <widget class="QLabel" name="appIcon">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="version">
     <property name="text">
      <string>Version info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="author">
     <property name="text">
      <string>Developed by Jan van Katwijk </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="buildInfo">
     <property name="text">
      <string>build info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="sourceLocation">
     <property name="text">
      <string>source location</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="qtVersion">
     <property name="text">
      <string>Based on Qt 5.2.2</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="libraries">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Qt-DAB uses following libraries:</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="configuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Qt-DAB is configured with :</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="acknowledgementsLabel">
       <property name="text">
        <string>Acknowledgements</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="disclaimer"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
